deepspeed/profiling/flops_profiler/profiler.py

```diff
@@ -280,11 +328,13 @@ def print_model_profile(self, profile_step=1, module_depth=-1, top_modules=1, de
             print('{:<60}  {:<8}'.format('data parallel size: ', self.ds_engine.dp_world_size))
             print('{:<60}  {:<8}'.format('model parallel size: ', self.ds_engine.mp_world_size))
             print('{:<60}  {:<8}'.format('batch size per GPU: ', self.ds_engine.train_micro_batch_size_per_gpu()))
+            if self.ds_engine.has_moe_layers:
+                print('{:<60}  {:<8}'.format('Expert parallelism: ', expert_parallelism))
```

Comment:
`expert parallel size` and let's move it below model parallel size. We should probably indicate whether `expert_tensor_parallelism` is enabled, since this means expert parameters parallelization is 3-dimensional.

Reply:
Not a big fan of expert parallel size but it's it's better than what it's there now. Will add expert tensor parallelism